packages/expo-task-manager/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### 💡 Others
 
+- [docs] Update background modes configuration for iOS in multiple SDK pages ([#32647](https://github.com/expo/expo/pull/32647) by [@chrfalch](https://github.com/chrfalch))
```

Comment:
We usually don't add changelog entries for docs changes, only for changes in the code of the package. This is because the docs are published separately from the package.
```suggestion
```

Reply:
Absolutely - was just following the suggestion from the bot!!! 😅